README.md

```diff
@@ -1788,6 +1789,43 @@ Map::from( ['a' => 'X', 'b' => 'Y'] )->has( 'X' );
 ```
 
 
+### if()
+
+Executes callables $then and $else depending on callable $condition.
+
+```php
+public function if( $condition, Callable $then, $Callable $else = null ) : self
+```
+* @param callable|bool $condition Bool or function with (Map) parameter which evaluates to bool
+* @param callable $then Function with (Map) parameter which returns void
+* @param callable|null $else Optional function with (Map) parameter which returns void
+* @return self Same map for fluid interface
+
+**Examples:**
+
+```php
+Map::from( ['a' => 1, 'b' => 0] )->if(
+    fn(Map $map) => $map->has('a'),
+    function(Map $_) {echo "then";},
+    function(Map $_) {echo "else";}
+);
+//then
+
+Map::from( ['a' => 1, 'b' => 0] )->if(
+    fn(Map $map) => $map->has('c'),
+    function(Map $_) {echo "then";},
+    function(Map $_) {echo "else";}
```

Comment:
Same here

Reply:
Resolved in 4e5cf55e0d52299bd0c86f499090473f0c5ab578